sbin/build.sh

```diff
@@ -475,12 +475,13 @@ makeACopyOfLibFreeFontForMacOSX() {
 
 
 # Get the first tag from the git repo
+# Excluding "openj9" tags as they have other ones for milestones etc. that get in the way
 getFirstTagFromOpenJDKGitRepo()
 {
     git fetch --tags "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}"
-    justOneFromTheRevList=$(git rev-list --tags --max-count=1)
-    tagNameFromRepo=$(git describe --tags "$justOneFromTheRevList")
-    echo "$tagNameFromRepo"
+    revList=$(git rev-list --tags --max-count=100)
```

Comment:
100 is a magic number can you extract that to a named const?

Reply:
Extracted into a variable in the constants file